libs/testing/containers/src/lib/testing-containers.ts

```diff
@@ -44,3 +46,23 @@ export const startRedisCluster = async () => {
 export const stopRedis = () => {
   redisClusterContainer.stop()
 }
+
+export const startLocalstack = async () => {
+  const lc = await new GenericContainer(
+    'public.ecr.aws/localstack/localstack:3',
+  )
+    .withName(`localstack-${Math.random().toString(16).slice(2, 8)}`)
+    .withExposedPorts(4566)
+    .withWaitStrategy(Wait.forLogMessage('Ready.'))
+    .start()
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  ;(global as any).__localstack__ = lc
```

Comment:
Maybe this would be worth a shot.

Adding this to the top of the file:

```typescript
declare global {
  // eslint-disable-next-line no-var
  var __localstack__: StartedTestContainer | undefined
}
```

Then here you could do:
```typescript
global.__localstack__ = lc
```

Reply:
Where would you put the global declaration?

Reply:
I put it right above its usage.